src/Build/BackEnd/Components/SdkResolution/SdkResolverRequest.cs

```diff
@@ -83,6 +89,7 @@ public void Translate(ITranslator translator)
             translator.Translate(ref _submissionId);
             translator.Translate(ref _version);
             translator.Translate(ref _interactive);
+            translator.TranslateDictionary(ref _globalProperties, count => new Dictionary<string, string>(count, StringComparer.OrdinalIgnoreCase));
```

Comment:
This will make node communication slower. Each node calls back into the main node when resolving sdks.